css/style.css

```diff
@@ -26,4 +27,43 @@ tr:nth-child(even), .highlight{
 
 #store-locations {
     text-indent: -10000px;
+}
+
+h2 {
+    margin-top: 2.3rem;
+    margin-left: 0.625rem;
+}
+
+form {
+    margin: 0.5rem; 
+}
+
+form {
+    width: 400px;
+    padding: 10px;
+    border: 1px solid #773344;
+}
```

Comment:
We have the `form` selector declared twice, on 41 and 37. We should combine these styles so it's easier to see everything being applied to a given selector in one spot.